pkg/translator/opencensus/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [open-telemetry/collector-approvers]
```

Comment:
This

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L132
```
pkg/translator/opencensus/                               @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
```